dev-tools/globals.ts

```diff
@@ -1,10 +1,13 @@
 type Battle = import('./../sim/battle').Battle
 type Field = import('./../sim/field').Field
 type ModdedDex = typeof import('./../sim/dex')
-type Pokemon = import('./../sim/pokemon').Pokemon
 type Side = import('./../sim/side').Side
 type Validator = import('./../sim/team-validator').Validator
 
+type Pokemon = import('./../sim/pokemon').Pokemon
+type RequestPokemonData = import('./../sim/pokemon').RequestPokemonData
```

Comment:
do `RequestPokemonData` and `RequestSubMoveSlot` work as `declare let Foo: import(...).Foo`?

Reply:
idk ;-;